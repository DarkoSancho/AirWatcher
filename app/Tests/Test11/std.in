1
government
2
Sensor10
no
no
no
no
20192-01-01
2019-01-01
2020-01-01
5
2