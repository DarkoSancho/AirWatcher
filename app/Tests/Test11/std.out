Welcome to AirWatcher App!

Menu:
1. Sign In
2. Exit
Select your choice: 
Enter user login: 
AirWatcher App -  Welcome to AirWatcher ! Let's get started.

AirWatcher App - Government Agency Menu 

1) - Calculate mean air quality over area.
2) - Show statistics for a specific sensor. 
3) - Score and rank sensors by similarity.
4) - Performance tests.
5) - Sign Out.

Select a functionality by choosing the corresponding number:

AirWatcher App - Showing all the readings of a Sensor
Enter Sensor ID (in the format SensorXX): 
Do you want to set a threshold for the attribute O3 ? (yes/no): 
Do you want to set a threshold for the attribute NO2 ? (yes/no): 
Do you want to set a threshold for the attribute SO2 ? (yes/no): 
Do you want to set a threshold for the attribute PM10 ? (yes/no): 
Enter Start Date in format YYYY-MM-DD : 
Enter End Date in format YYYY-MM-DD : 
---- Averages ----
NO2 : 71
O3 : 40
PM10 : 50
SO2 : 47
---- Minimum ----
NO2 : 67.98
O3 : 37.23
PM10 : 46.96
SO2 : 43.21
---- Maximum ----
NO2 : 76.03
O3 : 44.42
PM10 : 53.88
SO2 : 51.67
---- Standard deviation ----
NO2 : 1.72336
O3 : 1.67325
PM10 : 1.5073
SO2 : 1.49845
---- Exceedances ----
NO2 : no data / No threshold provided
O3 : no data / No threshold provided
PM10 : no data / No threshold provided
SO2 : no data / No threshold provided
---- Date and value ATMO min ----
12/01/2019 12:00:00 : 7
---- Date and value ATMO max ----
01/01/2019 12:00:00 : 8
NO2 : 71
O3 : 40
PM10 : 50
SO2 : 47

AirWatcher App - Government Agency Menu 

1) - Calculate mean air quality over area.
2) - Show statistics for a specific sensor. 
3) - Score and rank sensors by similarity.
4) - Performance tests.
5) - Sign Out.

Select a functionality by choosing the corresponding number:


AirWatcher App- You signed out successfully !

Menu:
1. Sign In
2. Exit
Select your choice: 
Exiting...
